packages/dashboard/src/Dashboard.tsx

```diff
@@ -0,0 +1,10 @@
+import { lazy, Suspense } from 'react'
+
+const Dashboard = lazy(() => import('./initialization/Dashboard'))
+export function IntergratedDashboard() {
+    return (
+        <Suspense fallback="">
```

Comment:
It's WIP?

Reply:
It's by design. This is used to defer the import module graph.